chapters/connectors.tex

```diff
@@ -920,7 +946,7 @@ \subsection{Generation of Connection Graph Equations}\label{generation-of-connec
 After this analysis, the \firstuse{connection graph equations}\index{connection equations!connection graph} are generated in the following way:
 \begin{enumerate}
 \item
-  For every optional spanning-tree edge (i.e., a \lstinline!connect(A, B)! equation), in one of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}.
+  For every optional spanning-tree edge (for \lstinline!connect(A, B)! with overdetermined connector \lstinline!R! this corresponds to the edge \lstinline!A.R! to \lstinline!B.R!), in one of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}.
```

Comment:
Avoid making it sound like just one of the spanning trees should be used:
```suggestion
  For every optional spanning-tree edge (for \lstinline!connect(A, B)! with overdetermined connector \lstinline!R! this corresponds to the edge \lstinline!A.R! to \lstinline!B.R!), in each of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}.
```

Reply:
I find that harder to read, and I think both variants are more complicated than necessary - and the underlying problem is that the parenthesis is disrupting the structure.
I would thus prefer something like:

`For every optional spanning-tree edge that belongs to a spanning tree (for \lstinline!connect(A, B)! with overdetermined connector \lstinline!R! this corresponds to the edge \lstinline!A.R! to \lstinline!B.R!), the connection equations are generated according to \cref{generation-of-connection-equations}.`
(Or possibly "any spanning tree".)

Reply:
Looking more I believe that "any spanning tree" is the best, as it corresponds to the next item. I will propose the specific text.

Reply:
Specifically:
```suggestion
  For every optional spanning-tree edge in any of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}.
  For \lstinline!connect(A, B)! with overdetermined connector \lstinline!R! this corresponds to the edge \lstinline!A.R! to \lstinline!B.R! generating the equation \lstinelin!A.R = B.R!.
```
This makes it closer to the next item.

Reply:
Sure, let's do that.